modules/administration-guide/examples/ref_che-supported-platforms-and-installation-methods.adoc

```diff
@@ -34,27 +31,27 @@ a|* `minikube`
 |OperatorHub
 |===
 
+WARNING: Setting up an OIDC provider is required for deploying {prod-short} on {kubernetes} infrastructures. For {ocp} no additional setup is needed.
```

Comment:
```suggestion
WARNING: Setting up [users authentication](https://kubernetes.io/docs/reference/access-authn-authz/authentication/) is required for deploying {prod-short} on {kubernetes} infrastructures. For {ocp} no additional setup is needed.
```

Reply:
Reading it the first time I thought the warning was about `Table 2. Local installation on OpenShift` (that comes just before). So I would say that it should be moved right after `Table 3. Public cloud installation on Kubernetes or OpenShift`, but I am not an expert. Take this comment with a grain of salt.